core/templates/dev/head/components/summary_tile/canonical_story_summary_tile_directive.html

```diff
@@ -0,0 +1,35 @@
+<a ng-href="<[getStoryLink()]>">
+  <div class="oppia-story-image">
+  </div>
+  <div class="oppia-story-details">
+    <div class="oppia-story-title">
+      <[getStoryTitle()]>
+    </div>
+    <div class="oppia-story-description">
+      <[getStoryDescription()]>
+    </div>
+  </div>
+</a>
+<style>
+  .oppia-story-image {
```

Comment:
Wrap all these styles in canonical-story-summary-tile to restrict to only this directive.

Reply:
Done.